src/components/checkbox/daikin-checkbox.ts

```diff
@@ -194,36 +170,25 @@ export class DaikinCheckbox extends LitElement {
   error = false;
 
   override render() {
-    // Specify the component size
-    const checkboxClassName = cvaCheckbox({ size: this.size });
-    const labelClassName = cvaLabel({ size: this.size });
-
-    const isIndeterminate = this.checkState === "indeterminate";
-
-    const labelText = this.label
-      ? html`<span class=${labelClassName}>${this.label}</span>`
-      : html``;
-    const inputTag = html`<input
-      class=${checkboxClassName}
-      type="checkbox"
-      name=${this.name}
-      value=${this.value}
-      aria-readonly=${this.readonly}
-      .indeterminate=${isIndeterminate}
-      .checked=${this.checked}
-      ?readonly=${this.readonly}
-      ?disabled=${this.disabled}
-      @change=${this._handleChange}
-      @click=${this._handleClick}
-    />`;
-    const content =
-      this.labelPosition === "left"
-        ? html`${labelText}${inputTag}`
-        : html`${inputTag}${labelText}`;
     return html`<label
-      class="inline-flex gap-[10px] items-center font-daikinSerif"
-      >${content}</label
-    >`;
+      class=${cvaContainer({ labelPosition: this.labelPosition })}
+    >
+      <input
+        class=${cvaCheckbox()}
+        type="checkbox"
+        name=${this.name}
+        value=${this.value}
+        aria-label=${this.labelPosition === "hidden" ? this.label : nothing}
+        .indeterminate=${this.checkState === "indeterminate"}
+        .checked=${this.checked}
+        ?disabled=${this.disabled}
+        @change=${this._handleChange}
+        @click=${this._handleClick}
+      />
+      ${this.labelPosition === "hidden"
+        ? nothing
+        : html`<span class="text-base">${this.label}</span>`}
```

Comment:
Maybe we should use `hidden`?

```suggestion
      <span class="text-base" ?hidden=${this.labelPosition === "hidden"}>${this.label}</span>
```